[SPARK-27261][DOC] Improve app submission doc for passing multiple configs

sujith71955 opened the pull request:
## What changes were proposed in this pull request?

While submitting the spark application, passing multiple configurations not documented clearly, no examples given.it will be better if it can be documented since  clarity is less from spark documentation side.
Even when i was browsing i could see few queries raised by users, below provided the reference.

https://community.hortonworks.com/questions/105022/spark-submit-multiple-configurations.html

 As part of fixing i had documented the above scenario  with an example.

## How was this patch tested?
Manual inspection of the updated document.

sujith71955 committed:
[SPARK-27261][Deploy] Documented regarding passing multiple configurations while submitting spark applications

## What changes were proposed in this pull request?

While submitting the spark application, passing multiple configurations not documented clearly, no examples given.it will be better if it can be documented since  clarity is less from spark documentation side.
documented the above scenario  with an example.

## How was this patch tested?
Manual inspection of the updated document.

AmplabJenkins commented:
Can one of the admins verify this patch?

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/testing-k8s-prb-make-spark-distribution-unified/9221/
Test PASSed.

SparkQA commented:
**[Test build #103855 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/103855/testReport)** for PR 24191 at commit [`0fee166`](https://github.com/apache/spark/commit/0fee166892f6a35d7f3d27472499c5df789a53c0).

SparkQA commented:
**[Test build #103855 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/103855/testReport)** for PR 24191 at commit [`0fee166`](https://github.com/apache/spark/commit/0fee166892f6a35d7f3d27472499c5df789a53c0).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/103855/
Test PASSed.

sujith71955 commented:
@cloud-fan @dongjoon-hyun Please let me know for your suggestions. thanks

HeartSaVioR commented on docs/submitting-applications.md:
```diff
@@ -44,7 +44,7 @@ Some of the commonly used options are:
 * `--master`: The [master URL](#master-urls) for the cluster (e.g. `spark://23.195.26.187:7077`)
 * `--deploy-mode`: Whether to deploy your driver on the worker nodes (`cluster`) or locally as an external client (`client`) (default: `client`) <b> &#8224; </b>
-* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown).
+* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown). Multiple configurations shall be passed individually.(e.g --conf <key>=<value> --conf <key>=<value>)
```
nit: `key2=value2` for second option

felixcheung replied:
also put `--conf <key>=<value> --conf <key>=<value` in backtick

srowen replied:
And you're missing a space after period, period after g, and the second sentence is all parenthetical. For minor changes, please do proofread them. Just: "Multiple configurations should be passed as separate arguments. Example: `--conf key1=value1 --conf key2=value2`.

sujith71955 replied:
Thanks all for review, will update the section as per the suggestions .

HeartSaVioR reviewed (approved):
LGTM, a nit.

dongjoon-hyun commented:
Thank you for pinging me, @sujith71955 . Yes, this will be helpful. Please revise according to the previous comments from @HeartSaVioR and @felixcheung .

HyukjinKwon reviewed (approved):
Looks good except the nit

sujith71955 committed:
[SPARK-27261][Deploy] Documented regarding passing multiple configurations while submitting spark applications

## What changes were proposed in this pull request?

While submitting the spark application, passing multiple configurations not documented clearly, no examples given.it will be better if it can be documented since  clarity is less from spark documentation side.
documented the above scenario  with an example.

## How was this patch tested?
Manual inspection of the updated document.

sujith71955 commented:
All comments has been addressed, let me know for any more clarifications/suggestions. Thanks all for your valuable time.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/testing-k8s-prb-make-spark-distribution-unified/9250/
Test PASSed.

SparkQA commented:
**[Test build #103887 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/103887/testReport)** for PR 24191 at commit [`5daa01a`](https://github.com/apache/spark/commit/5daa01ab9036e80c037226f8f256ba15db3091e4).

SparkQA commented:
**[Test build #103887 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/103887/testReport)** for PR 24191 at commit [`5daa01a`](https://github.com/apache/spark/commit/5daa01ab9036e80c037226f8f256ba15db3091e4).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/103887/
Test PASSed.

srowen reviewed (approved):
Close enough

dongjoon-hyun reviewed (approved):
+1, LGTM. Merged to master.
